src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -26,15 +25,17 @@ if (__DEV__) {
   };
   var warnedProperties = {};
 
-  var warnUnknownProperty = function(name) {
+  var warnUnknownProperty = function(name, source) {
     if (DOMProperty.properties.hasOwnProperty(name) || DOMProperty.isCustomAttribute(name)) {
       return;
     }
     if (reactProps.hasOwnProperty(name) && reactProps[name] ||
         warnedProperties.hasOwnProperty(name) && warnedProperties[name]) {
       return;
     }
-
+    if (EventPluginRegistry.registrationNameModules.hasOwnProperty(name)) {
```

Comment:
Because previously we were processing only non-event properties because we were listening in setValueForProperty or whatever, but now we are just reading off the element so we need to filter out the events ourselves.